rosetta/tests/upstream-only-distribution.sh

```diff
@@ -6,20 +6,37 @@ cd $SCRIPT_DIR
 set -eou pipefail
 
 # Version should work on Linux || Darwin
-repo_tmp=$(mktemp -d 2>/dev/null || mktemp -d -t 'mytmpdir')
-patchlist_tmp=$(mktemp /tmp/patchlist.txt.XXXXXX)
+tmp_base=$(mktemp -d 2>/dev/null || mktemp -d -t 'upstream')
+workspace_tmp=$(mktemp -d 2>/dev/null || mktemp -d -t 'workspace')
+manifest_tmp=$(mktemp /tmp/manifest.yaml.XXXXXX)
 
-UPSTREAM_URL=https://github.com/google-research/t5x.git
+LIBRARY=t5x
 # Commit was taken just before PR-1372
 DISTRIBUTION_BASE_REF=22117ce5a3606706ba9519ccdd77b532ad8ff7b2
-
-git clone $UPSTREAM_URL $repo_tmp
-echo "pull/1372/head" >> $patchlist_tmp
-
-bash ../create-distribution.sh \
-    -r $DISTRIBUTION_BASE_REF \
-    -d $repo_tmp \
-    -p $patchlist_tmp
+repo_tmp=$tmp_base/$LIBRARY
+
+cat <<EOF >> $manifest_tmp
+t5x:
+  url: https://github.com/google-research/t5x.git
+  mirror_url: https://github.com/nvjax-svc-0/t5x.git
+  tracking_ref: main
+  latest_verified_commit: $DISTRIBUTION_BASE_REF
+  mode: git-clone
+  patches:
+    pull/1372/head: null
+EOF
+bash ../../.github/container/get-source.sh --base-dir $tmp_base --manifest $manifest_tmp --library $LIBRARY
+
+cp ../../.github/container/create-distribution.sh $workspace_tmp/
```

Comment:
Same here

Reply:
So `create-distribution.sh` will create the patches alongside it, which will clutter your local clone with untracked files. I don't want the git source to change (since I'd need to `trap` to clean up), so I move it into a workspace dir.